vendor/engines/c2po/app/views/users/_new_user.html.haml

```diff
@@ -1,4 +1,8 @@
 .notice
   %p= "#{t('users.new_user.success')} #{link_to "#{@new_user} (#{@new_user.username})", facility_user_path(current_facility, @new_user)}.".html_safe
   %p= link_to t('users.new_user.add_to_payment_source'), facility_accounts_path(current_facility)
-  %p= link_to t('users.new_user.order_for'), facility_user_switch_to_path(current_facility, @new_user)
+
+  - if current_facility.single_facility?
```

Comment:
Actually, based on my understanding of the ticket, it sounds like this should link should just go away entirely. Users won't have a payment source at this point, but ordering on behalf of them is kind of useless.